README.md

```diff
@@ -721,6 +724,120 @@ async function main() {
 main();
 ```
 
+## Downscoped Client
+
+[Downscoping with Credential Access Boundaries](https://cloud.google.com/iam/docs/downscoping-short-lived-credentials) enables setting up a distinct set of permissions through downscoping with Credential Access Boundary which is an easier and more
+manageable approach for developers to have Identity and Access Management (IAM) permissions on a short-lived credential compared with creating a bunch of service accounts with different roles. Another reason to use downscoped credentials is to ensure tokens in flight always have the least privileges, e.g. Principle of Least Privilege.
```

Comment:
First sentence is confusing, re-word:

[Downscoping with Credential Access Boundaries](..) is used to restrict the Identity and Access Management (IAM) permissions that a short-lived credential can use for Cloud Storage.

I also wouldn't use wording like 'creating a bunch'. I think you can copy the Java/Python READMEs here.

Reply:
ok